geopmdpy/geopmdpy/pio.py

```diff
@@ -600,7 +600,7 @@ def start_batch_server(client_pid, signal_config, control_config):
         RuntimeError: Failure to start the batch server.
 
     """
-
+    global _dl
```

Comment:
I didn't think this would be necessary since we aren't assigning the `_dl` variable to a new target in this scope. Why is it needed?

Reply:
Perhaps you are right. It is unclear to me if _dl is modified when you calls to its attributes.  Perhaps this is actually not relevant to the global syntax. The pattern is followed in pio.py